src/formatters/junit_formatter.js

```diff
@@ -0,0 +1,34 @@
+const escapeHtml = require('escape-html');
+
+export default function JUnitFormatter(errorsGroupedByFile) {
+  const files = Object.keys(errorsGroupedByFile);
+
+  var xml = '<?xml version="1.0" encoding="utf-8"?>\n';
+
+  xml = xml + '<testsuites>\n';
+
+  files.forEach(file => {
+    var errorsCount = errorsGroupedByFile[file].length;
+
+    xml =
+      xml +
+      `  <testsuite name="${file}" time="0" tests="${errorsCount}" errors="${errorsCount}">\n`;
+
+    errorsGroupedByFile[file].forEach(error => {
+      xml =
+        xml +
+        `    <testcase name="Line ${error.locations[0].line}, Column ${error
```

Comment:
@mscharley eslint uses `name="org.eslint.RULE_NAME"` and stores line/column information with the error message below. Not sure if this is the right approach, I followed the way tslint did it.

Reply:
I believe that `testcase[name]` should be something that uniquely identifies the specific test that was run, but that's the only limitation. This seems reasonable but I'll let you know if something breaks when I try to import the XML results.